### PR TITLE
live query and replication change listener examples.

### PR DESCRIPTION
**Pull request by jamiltz**
This change is [Reviewable](https://reviewable.io/reviews/couchbase/couchbase-lite-net/883)

### CODE REVIEW

**Comment by jamiltz** [comment]
@borrrden I added some details in the second commit. But the replication isn't working (document count on SG is still 0). And the replication status is `Busy`. Can you give it a go?

**Comment by borrrden** [comment]
Replication is working fine on my end (59 rows found in SG and replication completes)

**Comment by borrrden** [comment]
Actually on Mac it does what you say...

**Comment by borrrden** [comment]
Oh the logs say why...you can get around this by using an IP address instead.  This will need to be fixed for db012

**Comment by borrrden** [comment]
Actually this needs to be fixed in the .NET Core runtime itself (and actually it already has).  I tried with .NET Core 2.0 Preview 1 and replication works on Mac.

File: docs/examples/api-walkthrough/api-walkthrough/Program.cs [hunk 1]
```diff
@@ -85,11 +86,11 @@ static void Main(string[] args)
             }
 
             // live query
-            var liveQuery = query.ToLive();
+            var liveQuery = query.ToLiveQuery();
```
Comment (jamiltz) [inline docs/examples/api-walkthrough/api-walkthrough/Program.cs]: For DB11, should this be `.ToLive();`?

File: docs/examples/api-walkthrough/api-walkthrough/Program.cs [hunk 2]
```diff
@@ -85,11 +86,11 @@ static void Main(string[] args)
...
             liveQuery.Changed += (sender, e) => {
                 Console.WriteLine($"Number of rows :: {e.Rows.Count}");
             };
-            liveQuery.Run();
+            liveQuery.Start();
```
Comment (jamiltz) [inline docs/examples/api-walkthrough/api-walkthrough/Program.cs]: For DB11, should this be `.Run();`?

**Comment by borrrden** [comment]
No, db011 is b0154 not b0157

**Comment by jamiltz** [comment]
👍

**Pull request merged**